src/components/tab-controls/tab-controls.scss

```diff
@@ -1,6 +1,10 @@
+.tabs-control-container {
```

Comment:
do we need to introduce a container here? 🤔 we would need to refactor the HTML in our codebases to have a container now that you are removing properties from `.tab-controls` and adding these here

perhaps your new component should handle how its tab-controls are displayed, not change ALL tab controls?

Reply:
The container was already there, thankfully. But I need to turn it to flex here, which makes it easier for me to change the layout in mobile and turn it into a dropdown and even fool around with the order. You'll see!

Reply:
oooh my bad! as long as it doesn't break `/events/whats-on` or the "Events and Tours" section of `/welcome` we should be good